### PR TITLE
fix up code formatting and typo

### PR DESCRIPTION
**Pull request by woniper**
- fix up MultiResourceItemWriter code formatting
- fix up typo

### CODE REVIEW

File: spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/AbstractCursorItemReader.java [hunk 1]
```diff
@@ -485,7 +485,7 @@ protected T doRead() throws Exception {
 
 	/**
 	 * Read the cursor and map to the type of object this reader should return. This method must be
-	 * overriden by subclasses.
+	 * override by subclasses.
```
Comment (benas) [inline spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/AbstractCursorItemReader.java]: This should be "overridden". I can do it while rebasing this PR.

**Comment by benas** [comment]
Hi, 

Thank you for this PR! I added a comment about a typo. Please review it and your PR will be ready to merge.

Kr,
Mahmoud

File: spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/AbstractCursorItemReader.java (hunk 1 above)
Comment (lkw1989) [reply spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/AbstractCursorItemReader.java]: ok. I fixed it.

**Comment by benas** [comment]
LGTM. rebased and merged as c2b0a208c3bb9fd99f4581235b683b12502f9432 .

@woniper Thank you for your contribution!

**Pull request closed without merging**